src/DynamoCoreWpf/Views/Preview/InfoBubbleView.xaml.cs

```diff
@@ -16,6 +16,7 @@
 using Dynamo.ViewModels;
 using Dynamo.Wpf.UI;
 using Dynamo.Wpf.Utilities;
+using Microsoft.VisualBasic.Logging;
```

Comment:
probably an intellisense issue. You should not need this

Reply:
Thank you @pinzart90 updated